src/mod.rs

```diff
@@ -52,3 +54,25 @@ pub static ACTOR_BUNDLES: Lazy<[ActorBundleInfo; 8]> = Lazy::new(|| {
         },
     ]
 });
+
+pub type BlockPair = (Cid, Vec<u8>);
+
+pub fn read_car_as_stream<R>(reader: CarReader<R>) -> impl Stream<Item = BlockPair>
```

Comment:
Oh, I see. Let's change `car_cmd.rs` to not use the build module.

Reply:
> Oh, I see. Let's change `car_cmd.rs` to not use the build module.

If I understand this correctly - the proposal is to use the new `CarStream` in `car_cmd.rs`, while maintaining the old logic in the build script?

When I look at this it seems to me that removing the workspace completely might have been premature. If we'd be able to have this shared logic in another package - we would easily be able to reuse it within the build script. Perhaps a good enough reason to re-introduce the workspace with at least two packages `forest-core` and `forest-util`, for example. That would definitely allow us to get rid of `mod.rs` as per Aatif's [suggestion](https://github.com/ChainSafe/forest/issues/3324).

Reply:
Another option is to assemble the bundle when there's a new network version rather than in the build script.

Reply:
> Another option is to assemble the bundle when there's a new network version rather than in the build script.

True, there's only one downside to it - anything that's not automated is prone to human error. 

I'm fine with either approach as long as we don't have to maintain the outdated build script logic.